Description: Adds .GNU-stack section header to disable executable stack flag
Author: James Addison <jay@jp-hosting.net>
Origin: https://github.com/nodejs/node/pull/37688
--- a/src/large_pages/node_text_start.S
+++ b/src/large_pages/node_text_start.S
@@ -1,3 +1,6 @@
+#if defined(__ELF__)
+.section .note.GNU-stack,"",@progbits
+#endif
 .text
 .align 0x2000
 .global __node_text_start
